docs/aqua-book/language/header/header.md

```diff
@@ -2,21 +2,21 @@
 
 An Aqua source file has a head and a body. The body contains function definitions, services, types, constants. The header manages what is imported from other files and what is exported.
 
-## Module
+## Aqua component
 
-By default, `.aqua` file exports and declares everything it contains. With `module` header you can describe the `.aqua` file's interface.
+By default, `.aqua` file exports and declares everything it contains. With `aqua` header you can describe the `.aqua` file's interface.
 
 ```aqua
--- Module expression may be only on the very first line of the file
-module ModuleName declares *
+-- `aqua` expression may be only on the very first line of the file
+aqua AquaName declares *
 ```
 
-`Module.Name` may contain dots.
+`Aqua.Name` may contain dots.
 
-`ModuleName` can be used as the module's name when this file is `use`d. In this case, only what is enumerated in `declares` section will be available. `declares *` allows you to declare everything in the file as the module interface.
+`AquaName` can be used as the aqua's name when this file is `use`d. In this case, only what is enumerated in `declares` section will be available. `declares *` allows you to declare everything in the file as the module interface.
 
 ```aqua
-module ModuleName declares CONST_NAME, ServiceName, MyType, fn
+aqua AquaName declares CONST_NAME, ServiceName, MyType, fn
```

Comment:
AquaName => FileName
fn => functionName, because you cannot really use `fn` in `declares`, right?
```suggestion
aqua FileName declares CONST_NAME, ServiceName, MyType, functionName
```